src/reporter/consts.ts

```diff
@@ -0,0 +1,3 @@
+import { join } from 'path';
+
+export const coverageDir = join(process.cwd(), process.env.PW_GQL_TEMP_DIR?? '.gql-coverage');
```

Comment:
```suggestion
export const coverageDir = join(process.cwd(), process.env.PW_GQL_TEMP_DIR ?? '.gql-coverage');
```

Is there existed configured formatter in the repository?

Reply:
not yet. You are welcome start configure linter or/and prettier.